src/style.css

```diff
@@ -1 +1,8 @@
 /* styles go here */
+.fieldset {
+  margin-bottom: 20px;
+}
+
+.form-field + .form-field {
```

Comment:
looks strange classname + classname

Reply:
In this case I just select every next element, then I give margin